source/release-notes/4-0-0-rc1.rst

```diff
@@ -0,0 +1,85 @@
+.. index:: Crafter CMS 4.0.0 rc1 Release Notes
+
+---------------------
+Crafter CMS 4.0.0 rc1
+---------------------
+
+Crafter CMS 4.0 rc1 requires Java 11
+
+^^^^^^^^^^^^
+New Features
+^^^^^^^^^^^^
+
+* New Studio UI |br|
+
+  - New React-based UI |br|
+  - New configuration mechanics of the UI that allow plugins to wire to almost every aspect of the experience. This opens the door for having different authoring experiences per site. |br|
+  - New In-Context Editing system, Experience Builder (XB): |br|
+
+    - Edit content in-place |br|
+    - Drag and drop components, assets to add/remove or rearrange your site (supports sites built with FreeMarker, React and can easily be extended to support Vue, Angular and other UI frameworks) |br|
+    - See :ref:`experience-builder` for more information |br|
+
+* Plugins! |br|
+
+  - New plugin system allowing developers to build plugins for a site (headless/templated) that provide:
+
+    - Authoring side extensions allowing content authors new tools to build the desired experience. This includes extending Crafter Studio’s functionality with server-side code, UI widgets, third-party integration, pre-baked content model items and more. Learn all about it :ref:`here <studio-plugins>`
```

Comment:
We should link to `Site Plugins` and not refer to the old mechanism unless we have to support older plugins.